build.sbt

```diff
@@ -109,10 +109,10 @@ lazy val java = project.in(file("faunadb-java"))
     testOptions += Tests.Argument(TestFrameworks.JUnit, "+q", "-v"),
     apiURL := Some(url("http://faunadb.github.io/faunadb-jvm/faunadb-java/api/")),
 
-    (javacOptions in doc) := Seq("-source", "1.7",
+    (javacOptions in doc) := Seq("-source", "1.7", "-Xdoclint:none",
```

Comment:
I think I forgot to link the issue. I'm sorry.
https://github.com/faunadb/sales-engineering/issues/195

There was a discussion there if we rather disable lint of not.

In my opinion, the link is complaining about things that doesn't necessarily help the developer to understand the API. Things like all the parameters description and types if `Obj(String k1, Expr v1, ....)`, or all the `<T>` parameters description.

I believe we should invest time producing nice readable java docs rather than type information that is easy inferred by the method signature.

Let me know you're thoughts.